python/ray/data/_internal/block_batching/iter_batches.py

```diff
@@ -149,7 +154,7 @@ def _async_iter_batches(
             stats=stats,
             batch_format=batch_format,
             collate_fn=collate_fn,
-            num_threadpool_workers=prefetch_batches,
+            num_threadpool_workers=gpu_prefetch_batches,
```

Comment:
We still want full prefetching for the format conversion right? Just not the final GPU loading step.